src/ui/common/src/components/LogViewer/index.tsx

```diff
@@ -15,7 +16,9 @@ const LogViewer: React.FC<Props> = ({ logs, err }) => {
 
   const [selectedTab, setSelectedTab] = useState(0);
   const emptyElement = (
-    <Box sx={{ p: 2, backgroundColor: 'gray.100' }}>Nothing to see here!</Box>
+    <Box sx={{ p: 2, backgroundColor: theme.palette.gray[100] }}>
```

Comment:
Seems our sx prop isn't working with colors.

I'm not entirely sure what's going on there, but could be something we reach out to the MUI team for. This issue started appearing a couple weeks back when the frontend build did not work in dev mode.

Reply:
Can you try importing useTheme from MUI and logging the color palette to make sure that the gray[100] key is there?

Could be an issue of spelling of gray vs grey.

```
import { useTheme } from '@mui/material/styles';
const muiTheme = useTheme();
console.log('theme from context: ', theme)
```

Reply:
![image](https://user-images.githubusercontent.com/30596854/217911806-d0be7fe2-50c7-43a3-99a2-82a2a6d70ecf.png)